exec/jdbc/src/test/java/org/apache/drill/jdbc/ConnectionTest.java

```diff
@@ -57,7 +61,9 @@ public static void setUpConnection() throws SQLException {
     // Connection--and other JDBC objects--on test method failure, but this test
     // class uses some objects across methods.)
     Driver.load();
-    connection = DriverManager.getConnection( "jdbc:drill:zk=local" );
+    Properties properties = new Properties();
+    properties.setProperty(OptionValidator.OPTION_DEFAULTS_ROOT + ExecConstants.CREATE_PREPARE_STATEMENT_TIMEOUT_MILLIS, "30000");
```

Comment:
I wonder, should we define a function for this pattern?

```
ExecConstants.bootDefaultFor(
    ExecConstants.CREATE_PREPARE_STATEMENT_TIMEOUT_MILLIS)
```

But, we use Java 7, so the static function must reside elsewhere. Actually, the function might exist; I seem to recall making the same comment in the PR that externalized the defaults.

Reply:
Also, since properties don't have to be strings, why is this property stored as a string when the value is an int?

Reply:
I made ExecConstants a file class with a private constructor and added the static method there.

Reply:
The Properties class only allows strings to be set as values. The conversion to an integer is done somewhere inside the DriverManager.